main.go

```diff
@@ -72,9 +74,10 @@ func main() {
 
 	// This is for recovering from panic.
 	defer func() {
-		if r := recover().(error); r != nil {
-			_ = logger.CaptureException(r, &sentry.EventHint{
-				OriginalException: r,
+		r := recover()
+		if r != nil {
+			_ = logger.CaptureException(r.(error), &sentry.EventHint{
+				OriginalException: r.(error),
 			},
 				nil)
```

Comment:
btw, ini mending gini aja biar ga bingungin

```suggestion
				OriginalException: r.(error),
			}, nil)
```

Reply:
di format sama go fmt

Reply:
gofmt nge-ignore ini, jadi kalo diratain kaya saranku bakal dibiarin

Reply:
ok